### DISCUSSION TITLE
Plans about Node adapter deprecation

### DISCUSSION THREAD

**Discussion by g11dev** [keycloak/keycloak#30991] [Adapter deprecation]
We are starting a mobile app with a NestJs backend and we would like to use keycloak for authentication and authorization.

We noticed that adapters are being deprecated, but the conversation around it isn't very active and we were wondering if there were any updates about it.

Would you recommend starting the project using keycloak-connect adapter or waiting for updates on the situation? 

Thanks for your time and work

**Comment by jonkoops** [comment]
No, it's been deprecated for a while, don't use it. Try and use a library from your ecosystem that can speak standard OAuth/OpenID Connect to interface with Keycloak instead.
